avoidance/src/common.cpp

```diff
@@ -226,6 +226,11 @@ float wrapAngleToPlusMinusPI(float angle) { return angle - 2.0f * M_PI_F * std::
 
 float wrapAngleToPlusMinus180(float angle) { return angle - 360.f * std::floor(angle / 360.f + 0.5f); }
 
+float angleDifference(float a, float b) {
+  float angle = a - b;
```

Comment:
To be safe, rather do 
```suggestion
  float angle = fmod(a - b, 360.f);
```

Reply:
done